packages/react-components/react-tabster/src/focus/focusVisiblePolyfill.ts

```diff
@@ -103,3 +102,7 @@ function alreadyInScope(el: HTMLElement | null | undefined): boolean {
 
   return alreadyInScope(el?.parentElement);
 }
+
+function browserSupportsFocusVisible() {
+  return CSS.supports('selector(:focus-visible)');
```

Comment:
I would like to hear from @mshoho that we can use `focus-visible` in supported browsers.

Reply:
@miroslavstastny focus-visible isn't fully sufficient. It cannot be triggered programmatically. And keyborg has things like detecting if the focus is moved by screen readers (and consequently enabling the keyboard navigation mode).

Reply:
There is focusVisible option: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus
but: https://caniuse.com/?search=focusVisible